src/Uno.UI/UI/Xaml/Controls/MenuFlyout/CommandingHelpers.cs

```diff
@@ -0,0 +1,223 @@
+			{
+				if (value != null)
+				{
+					var valueAsI = value;
+
+					IconSource valueAsIconSource;
+					IconSourceElement returnValueAsIconElement = new IconSourceElement();
+
+					valueAsIconSource = valueAsI as IconSource;
+
+					returnValueAsIconElement.IconSource = valueAsIconSource;
+
+					return returnValueAsIconElement;
+				}
+				else
+				{
+					return null;
+				}
+			}
+
+			public object ConvertBack(object value, Type targetType, object parameter, string language)
+			{
+				throw new NotImplementedException();
+			}
+		}
+
+		class KeyboardAcceleratorCopyConverter : IValueConverter
+		{
+			public object Convert(object value, Type targetType, object parameter, string language)
+			{
+				if (value != null)
+				{
+					object valueAsI = value;
+
+					IList<KeyboardAccelerator> valueAsKeyboardAccelerators;
+					var returnValueAsKeyboardAcceleratorCollection = new DependencyObjectCollection<KeyboardAccelerator>();
+
+					valueAsKeyboardAccelerators = valueAsI as IList<KeyboardAccelerator>;
+					int keyboardAcceleratorCount;
+
+					keyboardAcceleratorCount = valueAsKeyboardAccelerators.Count;
+
+					// Keyboard accelerators can't have two parents,
+					// so we'll need to copy them and bind to the original properties
+					// instead of assigning them.
+					// We set up bindings so that modifications to the app-defined accelerators
+					// will propagate to the accelerators that are used by the framework.
+					for (int i = 0; i < keyboardAcceleratorCount; i++)
+					{
+						KeyboardAccelerator keyboardAccelerator = valueAsKeyboardAccelerators[i];
+						KeyboardAccelerator keyboardAcceleratorCopy = new KeyboardAccelerator();
+
+						keyboardAcceleratorCopy.SetBinding(KeyboardAccelerator.IsEnabledProperty, new Binding { Path = "IsEnabled", Source = keyboardAccelerator });
+						keyboardAcceleratorCopy.SetBinding(KeyboardAccelerator.KeyProperty, new Binding { Path = "Key", Source = keyboardAccelerator });
+						keyboardAcceleratorCopy.SetBinding(KeyboardAccelerator.ModifiersProperty, new Binding { Path = "Modifiers", Source = keyboardAccelerator });
+						keyboardAcceleratorCopy.SetBinding(KeyboardAccelerator.ScopeOwnerProperty, new Binding { Path = "ScopeOwner", Source = keyboardAccelerator });
+					}
+
+					return returnValueAsKeyboardAcceleratorCollection;
+				}
+				else
+				{
+					return null;
+				}
+			}
+
+			public object ConvertBack(object value, Type targetType, object parameter, string language)
+			{
+				throw new NotImplementedException();
+			}
+		}
+
+		internal static void BindToLabelPropertyIfUnset(
+			 ICommand uiCommand,
+			 FrameworkElement target,
+			 DependencyProperty labelProperty)
+		{
+			string localLabel = null;
+			var localLabelAsI = target.ReadLocalValue(labelProperty);
+
+			if (localLabelAsI != null)
+			{
+				localLabel = localLabelAsI.ToString();
+			}
+
+			Console.WriteLine($"BindToLabelPropertyIfUnset: [{localLabelAsI}]/[{localLabel}]");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)